examples/client.cc

```diff
@@ -866,12 +866,17 @@ int Client::send_packet() {
     return NETWORK_ERR_OK;
   }
 
+  if (-1 == connect(fd_, &remote_addr_.su.sa, remote_addr_.len)) {
+    sendbuf_.reset();
+    std::cerr << "connect: " << strerror(errno) << std::endl;
+    return NETWORK_ERR_OK;
+  }
```

Comment:
I think connect(2) can be done just only once in run().  Does mac os require to call connect(2) every time a packet is sent?